Kvent/views.py

```diff
@@ -1,5 +1,54 @@
 from django.http import HttpResponse
+from django.contrib import messages
+from django.contrib.auth import authenticate, login, logout
+from django.contrib.auth.forms import UserCreationForm
+from django.http import HttpResponseRedirect
+from django.urls import reverse, reverse_lazy
+from django.views.generic import TemplateView, CreateView, DetailView, UpdateView, FormView
+from django.contrib.auth.models import User
+from django.views import generic, View
+from django.shortcuts import render, redirect, get_object_or_404
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth.forms import UserCreationForm
 
 
+
+# from .forms import UserRegistrationForm, EventForm, AddQuestion
+from .models import *
+import datetime
+
 def index(request):
-    return HttpResponse("Hello, world. You're at the polls index.")
\ No newline at end of file
+    return HttpResponse("Hello, world. You're at the polls index.")
+
+def login(request):
+    if request.method == "POST":
+        username = request.POST.get('username')
+        password = request.POST.get('password')
+        try:
+            user = User.objects.get(username = username)
+            if user.password == password:
+                return HttpResponse("Yess")
+            else:
+                return HttpResponse("pass Nooo")
+        except User.DoesNotExist:
+            return HttpResponse("user Nooo")
+    
+
+def display_login(request):
+    return render(request, 'registration/login.html')
```

Comment:
Actually, I think there is no need to write these two function, you just use `{{next}}` in the login template file to redirect it to your return URL